lib/plugins/aws/deployFunction/tests/index.js

```diff
@@ -162,20 +164,4 @@ describe('AwsDeployFunction', () => {
       });
     });
   });
-
-  describe('#cleanup()', () => {
```

Comment:
This method is now completely delegated to the Package superclass. Package already has tests for cleanup, and this original test was mocking out to a Package instance anyway.